src/components/auth/RecoverPassword.vue

```diff
@@ -48,6 +62,24 @@ export default class RecoverPassword extends Vue {
     ];
     this.submitText = this.$t('authPages.recoverPassword');
   }
+
+  /**
+   * Method recover user's password by email from form
+   */
+  private async recoverPassword(): Promise<void> {
+    const emailField = this.fields.find(field => field.name === 'email');
+    const email = emailField ? emailField.value : '';
+
+    try {
+      await this.$store.dispatch(RECOVER_PASSWORD, { email });
+      this.$router.push('/login');
```

Comment:
let's pass a get parameter and show success-message on login screen
<img width="1267" alt="image" src="https://user-images.githubusercontent.com/3684889/74586353-c0e57100-4ff7-11ea-88d7-a8ca6e45293d.png">